### PR TITLE
Design cleanup

### PR DESCRIPTION
**Pull request by lschatzkin**
a general mid-project cleanup of the front end styling, more to come

### CODE REVIEW

**Comment by esoterik** [comment]
this looks great!!

when I ran it locally, the `.link-div` on the homepage was the size of the whole page, so clicking anywhere took me to an action. it looks like this is the case for all of the action page links on the homepage

**Comment by lschatzkin** [comment]
Good spotting! Fixed on the home page, and am not seeing css for this on the action pages

File: db/schema.rb [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```
Comment (wioux) [inline db/schema.rb]: Can you revert all these changes? db/schema.rb shouldn't be updated in a styling update.

Comment (lschatzkin) [reply db/schema.rb]: Agh! Sorry.

Comment (wioux) [reply db/schema.rb]: We shouldn't delete this file, only undo the changes to it made in e6110236761222ec13c4cf14b5bbb94540872cbf.

**Comment by lschatzkin** [comment]
I've cleanup up/restored a few things to leave bootstrap alonge as much as possible, ready for re-review

**Comment by esoterik** [comment]
we lost text links to actions on the homepage at some point -- sorry for not catching it before! it's in L42 of `app/views/welcome/index.html.erb` -- right now it's just an `h3` but it should also be a link.

**Comment by lschatzkin** [comment]
> we lost text links to actions on the homepage at some point -- sorry for not catching it before! it's in L42 of `app/views/welcome/index.html.erb` -- right now it's just an `h3` but it should also be a link.

Good catch! I'll fix this now.

**Comment by lschatzkin** [comment]
Actually, I'm not seeing anything different b/t this branch and master...

**Comment by esoterik** [comment]
oh weird, i guess on master the `span.link-div` covers the whole div and makes it clickable. I don't think we should go back to that (I don't expect whole paragraphs to be links when I'm interacting with a site), but I do expect the title of the action to be clickable, so I think we should still add that link to the `h3`

**Comment by lschatzkin** [comment]
Yep, I remember the link.div weirdness now, I'll add the link to h3.

**Comment by lschatzkin** [comment]
I messed around with a few other things while I was in there.

**Comment by esoterik** [comment]
lgtm, merging!

**Pull request merged**